spec/features/openid_connect/openid_connect_spec.rb

```diff
@@ -1298,7 +1298,7 @@ def oidc_end_client_secret_jwt(prompt: nil, user: nil, redirs_to: nil)
     access_token = token_response[:access_token]
     expect(access_token).to be_present
     expect(decoded_id_token[:acr]).to eq(Saml::Idp::Constants::IAL_VERIFIED_ACR)
-    expect(decoded_id_token[:vot]).to eq(nil)
+    expect(decoded_id_token).not_to have_key(:vot)
```

Comment:
Minor: Could we remove all traces of `vot` by turning this into an expectation of _only the keys_ that we expect to be present?